src/setuptools_scm/_integration/setuptools.py

```diff
@@ -14,7 +14,7 @@ def read_dist_name_from_setup_cfg(
     parser = configparser.ConfigParser()
 
     if isinstance(input, (os.PathLike, str)):
-        parser.read([input])
+        parser.read([input], encoding="utf-8")
```

Comment:
based on encoding behaviour, would it make sense to use 

```
with open(input, encoding="utf-8", errors="replace") as fp:
   parser.read_file(fp)
```

Reply:
In my opinion, it's better to use defaults unless there's a reason not to use the defaults, and I'm not aware of any reason to use the defaults. And since [Setuptools doesn't replace](https://github.com/pypa/setuptools/blob/feed34f981cd7035d787e1cb4825ea58a423152d/setuptools/dist.py#L712), neither should `setuptools_scm`. If there are users currently depending on the current behavior on Windows only, their project wouldn't install with Setuptools anyway, so the simpler form is best.